src/components/common/SafeTokenWidget/index.tsx

```diff
@@ -16,7 +17,8 @@ import SafeTokenIcon from './safe_token.svg'
 
 import css from './styles.module.css'
 
-const CLAIMING_APP_NAME = '$SAFE Claiming App'
+const isStaging = !IS_PRODUCTION && !cgwDebugStorage.get()
```

Comment:
Good catch to include the debug toggle, it's easy to miss. We should probably move `cgwDebugStorage.get()` to the definition of `IS_PRODUCTION` itself.